lte/gateway/python/scripts/feg_hello_cli.py

```diff
@@ -22,10 +23,35 @@
 
 @cloud_grpc_wrapper
 def echo(client, args):
+    start_time = datetime.datetime.now()
     req = HelloRequest(greeting=args.msg, grpc_err_code=args.err_code)
-    print("request:\n", req)
+    print("- Request:\n", req)
     resp = client.SayHello(req)
-    print("resp: ", resp)
+    end_time = datetime.datetime.now()
+    print(f'- Response: {resp.greeting}')
```

Comment:
should this be inside the `if resp` condition?

Reply:
good catch,  I think i forgot to remove that